lua/lspconfig/configs.lua

```diff
@@ -139,6 +139,9 @@ function configs.__newindex(t, config_name, config_def)
           api.nvim_create_autocmd('BufReadPost', {
             pattern = fn.fnameescape(root_dir) .. '/*',
             callback = function(arg)
+              if #M.manager:clients() == 0 then
+                return true
+              end
```

Comment:
I was hoping there is a way to checking that the clients were force-stopped. Could there be an edge case where the client dies because of an error, which would then (unintentionally) remove this autocmd?

I guess we can try this.